cmd/image/qcow2ova/prep/templates.go

```diff
@@ -0,0 +1,171 @@
+package prep
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+)
+
+var setup = `#!/bin/bash
```

Comment:
any possibility to make some of the package versions and repositories variables ? We can do it for the next iteration

Reply:
yes, we should. Planning to do while migrating to that newer cloud-init package.